news/2022-06-30-community-contributors-2021.md

```diff
@@ -0,0 +1,101 @@
+
+It is said that this singular utility is responsible for the birth of an entirely new mapping style, an accolade highly supported by the absolute deluge of nominations he received from the mapping and modding community.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/leoflt.jpg)](https://osu.ppy.sh/users/3668779)
+
+A constant presence in the tournament scene for many years and recently brought aboard as part of the official osu! World Cup management crew, LeoFLT's tireless efforts largely speak for themselves. From handling programming and statistics-related tasks, to spreadsheets, website designs and much more, his proven technical prowess is only the tip of the iceberg.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/chillierpear.jpg)](https://osu.ppy.sh/users/9501251)
+
+A forefront designer, caster and organizer in the osu! community tournament scene long before he became a part of the official World Cup management crew in recent months, ChillierPear's contributions in the field span many different domains. His graphical work is widely believed to have encouraged a significant uptick in the overall quality expectations for many community tournaments, plus his dedication to casting and streaming have left him with a well-earned reputation amongst those who make osu!'s competitive scene tick.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/cavoeboy.jpg)](https://osu.ppy.sh/users/7361815)
+
+The host and organizer for the largest osu! convention/LAN/gathering in the world, cavoeboy’s contributions have been integral towards building a sense of worldwide community in osu!. Attended by creator, developer, mapper, and player alike, cavoe's osu! event (COE) represents a monumental amount of effort and work that often goes chronically underappreciated, with professional firms on massively larger budgets struggling to organize events to the quality and expectations that COE reaches on the regular.
+
+Not content with only this, cavoeboy has been involved in an entire swathe of major osu! tournaments such as Corsace (both Corsace Open and Corsace Closed) and Organized Test Tournament, making it very clear that being a cornerstone for awesome experiences in the community is more or less his way of life.
+
+---
+
+Times remain trying around much of the world in these past few years, which only makes us all the more grateful for those who continue to carry the osu! community upon their shoulders. We continue to grow in both size and scope as a community, and with more major developments flickering into life as the months pass, it is almost certain that we will soon be approaching a new era in osu!'s 15-year history.
+
+To everyone who helps make the osu! community more awesome, we thank you from the bottom of our hearts, and we look forward to sharing and honouring your accolades someday soon.
+
+—the osu!team
```

Comment:
![image](https://user-images.githubusercontent.com/36758269/176658457-58b7ad59-1451-4c7f-9610-d1885879d920.png)

```suggestion
—the osu! team
```

Reply:
to be fair,
![image](https://user-images.githubusercontent.com/9954349/176661308-815a0090-5281-43cf-8378-09cd3c4198da.png)

Reply:
well either this or that changes i suppose